src/Simulator/IncrementalRedstoneSimulator/ObserverHandler.h

```diff
@@ -0,0 +1,128 @@
+
+#pragma once
+
+#include "RedstoneHandler.h"
+
+
+
+
+
+class cObserverHandler : public cRedstoneHandler
+{
+public:
+
+	inline static bool IsOn(NIBBLETYPE a_Meta)
+	{
+		return (a_Meta & 0x8) == 0x8;
+	}
+
+	inline static Vector3i Getಠ_ಠOffset(NIBBLETYPE a_Meta)
+	{
+		return -GetButtOffset(a_Meta);
+	}
+
+	inline static Vector3i GetButtOffset(NIBBLETYPE a_Meta)
+	{
+		switch (a_Meta & 0x7)
```

Comment:
This should be added as a static method on the observer BlockHandler: GetDirectionFromMeta